package.json

```diff
@@ -96,6 +97,7 @@
     "joi": "^17.6.0",
     "jsonwebtoken": "^8.5.1",
     "lodash": "^4.17.21",
+    "qrcode": "^1.5.0",
```

Comment:
Ina future iteration I'd like to remove one of the QR libraries. Doesn't make sense to be using two